DataConnectionCount test failing

```
18: =========================================================================== FAILURES ===========================================================================
18: _______________________________________________ DataConnectionCountTest.test_60_threads_vs_data_connection_count _______________________________________________
18: 
18: self = <tests.system_tests_one_router.DataConnectionCountTest testMethod=test_60_threads_vs_data_connection_count>
18: 
18:     def test_60_threads_vs_data_connection_count(self):
18:         for worker_threads in ('1', '3', '5'):
18:             for con_count in ('auto', '', '0', '2', '4'):
18:                 # Make the config file.
18:                 self.config[0][1]['workerThreads'] = worker_threads
18:                 if con_count == '':
18:                     del self.config[0][1]['workerThreads']
18:                 else:
18:                     self.config[0][1]['dataConnectionCount'] = con_count
18:     
18:                 # Figure out what log message we expect to see.
18:                 if con_count in ('auto', ''):
18:                     # This expression should match the one in
18:                     # the C function auto_calc_connection_count()
18:                     # in connection_manager.c .
18:                     expected_result = int((int(worker_threads) + 1) / 2)
18:                     msg = "Inter-router data connections calculated at " + str(expected_result)
18:                 else:
18:                     expected_result = con_count
18:                     msg = "Inter-router data connections set to " + str(expected_result)
18:     
18:                 print("worker threads:", worker_threads, ", requested connections:", con_count, ", expected result:", expected_result, end=" ")
18: >               self.router = self.tester.qdrouterd(self.name, self.config)
18: 
18: /home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_tests_one_router.py:3595: 
18: _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
18: /home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_test.py:1377: in qdrouterd
18:     return self.cleanup(Qdrouterd(*args, **kwargs))
18: /home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_test.py:792: in __init__
18:     self.wait_ready()
18: /home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_test.py:1094: in wait_ready
18:     self.wait_connectors(**retry_kwargs)
18: _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
18: 
18: self = <Qdrouterd: returncode: None args: ['skrouterd', '-c', 'dcc-test-router.conf...>, retry_kwargs = {}
18: c = {'host': '127.0.0.1', 'idleTimeoutSeconds': '120', 'port': 26380, 'role': 'inter-router', ...}
18: @py_assert1 = <function Qdrouterd.wait_connectors.<locals>.<lambda> at 0xffc3e98f59e0>, @py_assert4 = None
18: @py_format6 = 'Port not connected 26380\n>assert None\n{None = retry(<function Qdrouterd.wait_connectors.<locals>.<lambda> at 0xffc3e98f59e0>, **{})\n}'
18: 
18:     def wait_connectors(self, **retry_kwargs):
18:         """
18:         Wait for all connectors to be connected
18:         @param retry_kwargs: keyword args for L{retry}
18:         """
18:         for c in self.config.sections('connector'):
18: >           assert retry(lambda c=c: self.is_connected(port=c['port'], host=c.get('host') if c.get('host') else self.get_host(c.get('socketAddressFamily'))),
18:                          **retry_kwargs), "Port not connected %s" % c['port']
18: E           AssertionError: Port not connected 26380
18: E           assert None
18: E            +  where None = retry(<function Qdrouterd.wait_connectors.<locals>.<lambda> at 0xffc3e98f59e0>, **{})
18: 
18: /home/runner/work/skupper-router/skupper-router/skupper-router/tests/system_test.py:1058: AssertionError
18: ---------------------------------------------------------------------- Captured log call -----------------------------------------------------------------------
18: INFO     proton:_reactor.py:1040 Connecting to Url('amqp://0.0.0.0:26380')...
18: INFO     proton:_reactor.py:1078 Connected to 0.0.0.0
18: INFO     proton:_reactor.py:1092 Disconnected, already closed
18: INFO     proton:_reactor.py:1040 Connecting to Url('amqp://0.0.0.0:26380')...
18: INFO     proton:_reactor.py:1092 Disconnected, already closed
18: ------------- generated xml file: /home/runner/work/skupper-router/skupper-router/skupper-router/build/tests/junitxmls/system_tests_one_router.xml -------------
18: =================================================================== short test summary info ====================================================================
18: FAILED ::DataConnectionCountTest::test_60_threads_vs_data_connection_count - AssertionError: Port not connected 26380
18: assert None
18:  +  where None = retry(<function Qdrouterd.wait_connectors.<locals>.<lambda> at 0xffc3e98f59e0>, **{})
18: ERROR ::DataConnectionCountTest::test_60_threads_vs_data_connection_count - RuntimeError: Errors during teardown: 
18: 
18: Timeout: Connection amqp://0.0.0.0:26380 timed out: Opening connection
18:  None
18: ====================================================== 1 failed, 47 passed, 1 error in 663.21s (0:11:03) =======================================================
38/38 Test #18: system_tests_one_router ...............***Failed  664.62 sec

```

https://github.com/skupperproject/skupper-router/actions/runs/14409509432/job/40416855725?pr=1780#step:9:3790

--- a/tests/system_tests_one_router.py
+++ b/tests/system_tests_one_router.py
@@ -3558,8 +3558,7 @@ class DataConnectionCountTest(TestCase):
         DataConnectionCountTest.listen_port = cls.tester.get_port()
         cls.config = Qdrouterd.Config([
             ('router', {'mode': 'interior', 'id': 'DCC', 'workerThreads': '7'}),
-            ('listener', {'port': DataConnectionCountTest.listen_port, 'role': 'normal'}),
-            ('connector', {'role': 'inter-router', 'host': '127.0.0.1', 'port': DataConnectionCountTest.listen_port, 'saslMechanisms': 'ANONYMOUS', 'idleTimeoutSeconds': '120'}),
+            ('listener', {'port': DataConnectionCountTest.listen_port, 'role': 'normal'})
         ])
         cls.name = "dcc-test-router"
 

